content/en/docs/kubernetes/collector/components.md

```diff
@@ -13,15 +13,167 @@ enhancing it.
+        key: label1
+        from: namespace
+      - tag_name: l2 # extracts the value of a pod label with key `label2` with regexp and inserts it as a resource attribute with key `l2`
+        key: label2
+        regex: field=(?P<value>.+)
+        from: pod
+  pod_association: # how to associate the data to a pod (order matters)
+    - sources: # First try to use the value of the resource attribute k8s.pod.ip
+        - from: resource_attribute
+          name: k8s.pod.ip
+    - sources: # Then try to use the value of the resource attribute k8s.pod.uid
+        - from: resource_attribute
+          name: k8s.pod.uid
+    - sources: # If neither of those work, use the request's connection to get the pod IP.
+        - from: connection
+```
+
+There are also special configuration options for when the collector is deployed
+as a Kubernetes DaemonSet (agent) or as a Kubernetes Deployment (gateway). For
+details, see
+[Deployment Scenarios](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor#deployment-scenarios)
+
+For Kubernetes Attributes Processor configuration details, see
+[Kubernetes Attributes Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor).
+
+Since the processor uses the Kubernetes API, it needs the correct permission to
+work correctly. For most use cases, you should give the service account running
+the collector the following permissions via a ClusterRole.
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: collector
+  namespace: <OTEL_COL_NAMESPACE>
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - 'pods'
+      - 'namespaces'
+    verbs:
+      - 'get'
+      - 'watch'
+      - 'list'
+  - apiGroups:
+      - 'apps'
+    resources:
+      - 'replicasets'
+    verbs:
+      - 'get'
+      - 'list'
+      - 'watch'
+  - apiGroups:
+      - 'extensions'
+    resources:
+      - 'replicasets'
+    verbs:
+      - 'get'
+      - 'list'
+      - 'watch'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector
+subjects:
+  - kind: ServiceAccount
+    name: collector
+    namespace: <OTEL_COL_NAMESPACE>
+roleRef:
+  kind: ClusterRole
+  name: otel-collector
+  apiGroup: rbac.authorization.k8s.io
+```
+
 ## Filelog Receiver
 
+| Deployment Pattern   | Usable                                                          |
+| -------------------- | --------------------------------------------------------------- |
+| DaemonSet (Agent)    | Preferred                                                       |
+| Deployment (Gateway) | Yes, but will only collect logs from the node it is deployed on |
+| Sidecar              | Yes, but this would be considered advanced configuration        |
+
```

Comment:
Same remark as above, can we move this table to be somewhere other than immediately after the heading?

Reply:
Leave as is, as previously agreed.